pkg/ottl/ottlfuncs/README.md

```diff
@@ -238,6 +239,27 @@ Examples:
 
 - `limit(resource.attributes, 50, ["http.host", "http.method"])`
 
+### merge_maps
+
+`merge_maps(target, source)`
```

Comment:
Why not just "merge" and say for the moment only works with maps?

Reply:
I can try this out.  Is slice the only other type that we'd support in a merge function?

Reply:
@bogdandrutu updated the name to merge, but with that addition of the `strategy` param, is there another type that you think we'll be supporting in a merge function in the future?